ocaml/idl/datamodel_host.ml

```diff
@@ -979,6 +979,16 @@ let host_query_ha = call ~flags:[`Session]
       ~allowed_roles:_R_POOL_ADMIN
       ()
 
+  let emergency_server_certificate_reset = call
+      ~flags:[`Session]
+      ~in_oss_since:None
+      ~lifecycle:[Published, rel_stockholm, ""]
+      ~name:"emergency_server_certificate_reset"
```

Comment:
I'd say `emergency_reset_server_certificate`, for consistency with `install_server_certificate` and `get_server_certificate`. Although I know that some of the other emergency commands have the verb at the end... :(

Reply:
😵

Reply:
should I change the CLI name as well?

Reply:
It's better to make it consistent...

Reply:
I think having inconsistent names would be consistent as well ;p